forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -45,6 +45,7 @@ public class GitHubPullRequest implements PullRequest {
     private final Logger log = Logger.getLogger("org.openjdk.skara.host");
 
     private List<Label> labels = null;
+    private Optional<Boolean> diffLimited = Optional.empty();
```

Comment:
We should not use `Optional` in fields. If we need a tri-state for this boolean, then it's better to use `Boolean` and check for null.

Reply:
Will fix it